src/sentry/api/endpoints/project_release_files.py

```diff
@@ -130,6 +152,73 @@ def post_releasefile(request, release, logger):
         return Response(serialize(releasefile, request.user), status=201)
 
 
+class ListQuerySet:
```

Comment:
I think it would be better to write a custom paginator before doing this

Reply:
Turns out `ChainPaginator` already does what I need, using that now.